### ISSUE TITLE
Flutter Web Behavior

### ISSUE THREAD

**Issue by abbadabbatech** [jonbhanson/flutter_native_splash#253]
When I compile for web in debug mode, the splash works as expected.

However when I do release build and run from a webserver, the splash displays, but then it overlays
over the web app when it loads.

Any Idea why this would happen of how to fix it?

Thanks

**Comment by jonbhanson** [comment]
The package places tags in your `web/index.html` file, just above the closing `</body>` tag:

```html
<picture id="splash">
    <source srcset="splash/img/light-1x.png 1x, splash/img/light-2x.png 2x, splash/img/light-3x.png 3x, splash/img/light-4x.png 4x" media="(prefers-color-scheme: light) or (prefers-color-scheme: no-preference)">
    <source srcset="splash/img/dark-1x.png 1x, splash/img/dark-2x.png 2x, splash/img/dark-3x.png 3x, splash/img/dark-4x.png 4x" media="(prefers-color-scheme: dark)">
    <img class="center" aria-hidden="true" src="splash/img/light-1x.png" />
</picture>
```

Move these tags from their current location to just below the opening `<body ... >` tag.

Sorry for the trouble, I will fix it in the next release.

**Comment by jonbhanson** [comment]
Fixed in 1.3.3
